OOPConsoleApp/OOP/OOP.cs

```diff
@@ -0,0 +1,266 @@
+﻿using OOP.Interface;
+
+namespace OOP
+{
+    public static class OOP
+    {
+        private static void Main()
+        {
+            Exercise1();
+            Exercise2();
+            Exercise3();
+            Exercise4();
+            Exercise5();
+
+            void Exercise1()
+            {
+                Console.WriteLine("Exercise 1.");
+
+                var person = new Person("Fleming", "Lise", 33);
+
+                Console.WriteLine($"{person.GetSurname()} " +
+                    $"{person.GetName()}, " +
+                    $"{person.GetAge()}");
+            }
+
+            void Exercise2()
+            {
+                Console.WriteLine("Exercise 2.");
+
+                var student = new Student("Koval", "Irina", 19);
+                student.SetStudentAbility("musical ear");
+                student.SetStudentAssesment(5);
+
+                Console.WriteLine($"Student - {student.Surname}" +
+                    $" {student.Name}," +
+                    $" {student.Age} years old.");
+                Console.WriteLine($"Ability: {student.StudentAbility}. Assesment: {student.StudentAssesment}.");
+
+                var teacher = new Teacher("Ivanov", "Ivan", 41);
+                teacher.SetTeacherTemperament("sanguine");
+                teacher.SetTeacherSubject("Mathematicks");
+
+                Console.WriteLine($"Teacher - {teacher.PersonSurname} " +
+                    $"{teacher.PersonName}, " +
+                    $"{teacher.PersonAge} years old. ");
+                Console.WriteLine($"Temperament: {teacher.TeacherTemperament}. Subject: {teacher.TeacherSubject}.");
+            }
+            void Exercise3()
+            {
+                Console.WriteLine("Exercise 3.");
+
+                var personSecond = new Person("Datchin", "Valery", 43);
+
+                Console.WriteLine($"{personSecond.GetSurname()} {personSecond.GetName()}," +
+                $" {personSecond.GetAge()} y.o.");
+            }
+            void Exercise4()
+            {
+                Console.WriteLine("Exercise 4.");
+
+                var myPerson = new Person("Fleming", "Lise", 33);
+                myPerson.Print();
+
+                var myStudent = new Student("Koval", "Irina", 19);
+                myStudent.Print();
+
+                var myTeacher = new Teacher("Ivanov", "Ivan", 41);
+                myTeacher.Print();
+            }
+
+            void Exercise5()
+            {
+                Console.WriteLine("Exercise 5.");
+
+                var classroom = new Classroom();
+                classroom.AddPersons();
+                classroom.Print();
+                classroom.RemovePersons();
```

Comment:
The point of the method is to remove any person (student or teacher) from the class through parameters, using the same method in these different cases.
When deleting, it makes sense to use some kind of identifier in the parameters.